### ISSUE TITLE
Compatability issue with LC_API and MoreCompany

### ISSUE THREAD

**Issue by SirzBenjie** [notnotnotswipez/MoreCompany#36]
I have a friend that is unable to join my game and we have isolated it to this mod.  When we have both this mod and LC_API installed, he is unable to join my lobby.  When we have just one or just the other, he can join just fine.

This is the error from my end.
```
[Warning: Unity Log] [Netcode] NetworkConfig mismatch. The configuration between the server and client does not match  
[Info   : Unity Log] Disconnect callback called
[Info   : Unity Log] Is server: True; ishost: True; isConnectedClient: True
[Info   : Unity Log] Disconnect callback called in gamenetworkmanager; disconnecting clientId: 1
[Info   : Unity Log] A Player disconnected but they were not in clientplayerlist
[Info   : Unity Log] PLAYER OPENED MENU
[Info   : Unity Log] In terminal menu: False
```

**Comment by notnotnotswipez** [comment]
MoreCompany does not require LC_API

**Comment by notnotnotswipez** [comment]
Also, that error is because theres some network configurator mod thats mismatching your connections. Make sure you both have exactly the same mods installed

**Comment by SirzBenjie** [comment]
> MoreCompany does not require LC_API

I know it doesn't require it, other mods we have do, though.
I also know that this error is why this occurs.

We installed mods one at a time from scratch to isolate which mods were conflicting.  We have a profile on thunderstore mod manager with the exact same mods.
We tried several methods, copying cache, clearing cache, sharing code, installing the exact same versions, etc.

We reduced this to a minimum working example that demonstrates the bug.  Include _just_ MoreCompany, BepinEx, and LC_API. These were the only mods we have installed, and this error occurs.
